Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -1279,6 +1279,22 @@
             },
             "showPin": true,
             "name": "controlplaneerrors"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "alertsmanagementresources\r\n| where type == 'microsoft.alertsmanagement/alerts'\r\n| extend severity = tostring(properties[\"essentials\"][\"severity\"])\r\n| extend startTime = properties[\"essentials\"][\"startDateTime\"]\r\n| extend description = properties[\"essentials\"][\"description\"]\r\n| where properties[\"essentials\"][\"targetResource\"] in~ (\"{ase}\", \"{pccp}\", \"{kubernetesCluster}\")\r\n| where properties[\"essentials\"][\"monitorCondition\"] in~ ('Fired')\r\n| project [\"Alert Time\"]=todatetime(startTime),[\"Severity\"]=severity,[\"Alert\"]=name,[\"Description\"]=description\r\n| order by [\"Alert Time\"] desc",
+              "size": 0,
+              "title": "All Alerts",
+              "noDataMessage": "No active alerts",
```

Comment:
You should set the "No data message style" advanced setting to "Success" - to make it show up as a success message when there are no alerts.